base/shell/explorer/traywnd.cpp

```diff
@@ -307,6 +345,8 @@ class CTrayWindow :
     SIZE m_AutoHideOffset;
     TRACKMOUSEEVENT m_MouseTrackingInfo;
 
+    enum WndArrangement m_Arrangement;
```

Comment:
I wonder why the arrangement is a member of the CTrayWindow, but the backup storage is a global.
Wouldn't it be more logical to have either both being globals, or both being members?

Reply:
My logic was to place the backup storage as global, as it is already done of the list of g_MinimizedAll. Meanwhile, the selected Arrangnement as a state seems to fits logically as a member of CTrayWindow. But if you consider it's a MUST to gather both on one side on the other, just tell me.

Reply:
If the selected arrangement was a member of the ctraywindow, this would mean, that if you had two (or more) ctraywindow objects existing, you could manage at the same time two different arrangements, while at the same time you could manage only one backup.... For me it doesn't make much sense :thinking:

Reply:
That's definitively a good point. Arrangement has to move to global.